### ISSUE TITLE
Hydra fails to start: invalid example .env values

### ISSUE THREAD

**Issue by samkelleher** [reactioncommerce/reaction-hydra#16]
Type: breaking

**Describe the bug**
Following merge of #15 it's not possible to create a Hydra instance using `docker-compose` commands or the `make` commands in reaction-platform.

**To Reproduce**
Run `make init reaction-hydra` from reaction-platform.
Or, run `docker-compose up -d` from within this repo.

**Expected behavior**
The Hydra server runs normally.

**Observerd behavior**
Hydra container fails to start, logged out in the terminal before termination.

> level=fatal msg="The pairwise subject identifier algorithm is not supported by the JWT OAuth 2.0 Access Token Strategy. Please remove \"pairwise\" from OIDC_SUBJECT_TYPES_SUPPORTED or set OAUTH2_ACCESS_TOKEN_STRATEGY to \"opaque\""

**Comment by ticean** [comment]
@samkelleher can you paste the contents of the `.env` file that's currently present in your `reaction-hydra` directory? (Please ensure that it doesn't contain any secrets before posting. This will be the case if you haven't modified it.)

**Comment by samkelleher** [comment]
@ticean  It's just the regular `.env.example` copied to `.env` 

```
DATABASE_URL=postgres://hydra:changeme@postgres:5432/hydra?sslmode=disable
OAUTH2_ACCESS_TOKEN_STRATEGY=jwt
OAUTH2_CONSENT_URL=http://localhost:3000/consent
OAUTH2_ISSUER_URL=http://localhost:4444
OAUTH2_LOGIN_URL=http://localhost:3000/login
OAUTH2_SHARE_ERROR_DEBUG=1
OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
SYSTEM_SECRET=youReallyNeedToChangeThis
```

Removing the `pairwise` resolved the issue for me (in that it's able to start) but I don't know what consequences that has yes.

**Comment by ticean** [comment]
Thanks. The `OAUTH2_ACCESS_TOKEN_STRATEGY=jwt` shouldn't be there or should be set to `opaque`. The [`docker-compose.yml` history](https://github.com/reactioncommerce/reaction-hydra/commit/028ad39af43b0cd1232c3a2a1af4239e087308f5#diff-4e5e90c6228fd48698d074241c2ba760) shows that the config was commented in the file before it was moved to the `.env.example`.

You can change that on  your local at your leisure. I'll open an issue or PR to get it fixed in the project.

Thanks for reporting!

**Comment by focusaurus** [comment]
That's probably my fault in my recent change related to this.

**Comment by ticean** [comment]
Hey @focusaurus. I did a quick fix, too, thinking it would be faster to fix than to document. :) It explicitly sets the value to `opaque` rather than commenting.

**Comment by focusaurus** [comment]
OK I'll defer to you and Seun on this since I'm mostly in the cleaning staff role here. Also yaml comments where the hash sign is way on the left and the value is way indented are easy to miss in git diffs.

**Comment by ticean** [comment]
Thanks @focusaurus. I was working quickly yesterday and thought that the commented value in #17 was `jwt`. I looked again and see it's `opaque`. So, a commented var with default value is the winner here!

I verified that `opaque` is indeed the default. I couldn't quickly find the code that sets the default in the Hydra codebase, but here's [a statement in the changelog](https://github.com/ory/hydra/blob/d88fb128f26793e2f313c63ede0906782280d9b9/UPGRADE.md#json-web-token-formatted-access-token-data) that verifies.
